pom.xml

```diff
@@ -66,7 +67,8 @@ SOFTWARE.
   </issueManagement>
   <scm>
     <connection>scm:git:git@github.com:l3r8yJ/prefixed-maven-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:l3r8yJ/prefixed-maven-plugin.git</developerConnection>
+    <developerConnection>scm:git:git@github.com:l3r8yJ/prefixed-maven-plugin.git
+    </developerConnection>
```

Comment:
@l3r8yJ can we move this to the previous line?

Reply:
@h1alexbel it's longer than 100 chars, I'll split it